test/ad.jl/ad.jl

```diff
@@ -4,6 +4,7 @@ using Turing: VarInfo, gradient_forward, gradient_reverse
 @model foo_ad() = begin
     x ~ Normal(3, 1)
     y ~ Normal(x, 1)
+    return x, y
```

Comment:
A return statement is not mandatory in Turing. Or is it now?

Reply:
No I don't think so. I will revert this.